MobileCenter/MobileCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -1,118 +1,182 @@
 #import "MSDBStoragePrivate.h"
 #import "MSDatabaseConnection.h"
 #import "MSLogger.h"
+#import "MSMobileCenterInternal.h"
 #import "MSSqliteConnection.h"
-
-static NSString *const kMSLogEntityName = @"MSDBLog";
-static NSString *const kMSDBFileName = @"MSDBLogs.sqlite";
-static NSString *const kMSLogTableName = @"MSLog";
-static NSString *const kMSGroupIdColumnName = @"groupId";
-static NSString *const kMSDataColumnName = @"data";
+#import "MSUtility.h"
 
 @implementation MSDBStorage
 
-@synthesize connection;
-
 #pragma mark - Initialization
 
 - (instancetype)init {
   self = [super init];
   if (self) {
-    self.connection = [[MSSqliteConnection alloc] initWithDatabaseFilename:kMSDBFileName];
+    _connection = [[MSSqliteConnection alloc] initWithDatabaseFilename:kMSDBFileName];
+    _batches = [NSMutableDictionary<NSString *, NSArray<NSString *> *> new];
```

Comment:
Why does the storage need to pay attention to batch things? This is a log manager role.